src/Objectives/objs.jl

```diff
@@ -0,0 +1,2 @@
+include("elbo.jl")
+include("loglikelihood.jl")
```

Comment:
Rename the folder to `src/objectives/` (use lower-case, unless you're actually defining a `module` in there, and then make remame this file to `src/objectives.jl` and do

```suggestion
include("objectives/elbo.jl")
include("objectives/loglikelihood.jl")
```